src/procedures/__tests__/CreateCheckpoint.ts

```diff
@@ -17,21 +23,36 @@ const params1 = {
 
 describe('CreateCheckpoint', () => {
   let target: CreateCheckpoint;
+
+  let securityTokenMock: MockManager<contractWrappersModule.SecurityToken_3_0_0>;
+  let findEventsStub: SinonStub<any, any>;
+
   let contextMock: MockManager<contextModule.Context>;
   let wrappersMock: MockManager<wrappersModule.PolymathBase>;
   let tokenFactoryMock: MockManager<tokenFactoryModule.MockedTokenFactoryObject>;
   let tokenFactoryMockStub: SinonStub<any, any>;
 
-  beforeAll(() => {
+  // Mock factories
+  let checkpointFactoryMock: MockManager<checkpointFactoryModule.CheckpointFactory>;
+
+  beforeEach(() => {
     // Mock the context, wrappers, and tokenFactory to test CreateCheckpoint
     contextMock = ImportMock.mockClass(contextModule, 'Context');
     wrappersMock = ImportMock.mockClass(wrappersModule, 'PolymathBase');
     tokenFactoryMock = ImportMock.mockClass(tokenFactoryModule, 'MockedTokenFactoryObject');
-
-    tokenFactoryMockStub = tokenFactoryMock.mock('getSecurityTokenInstanceFromTicker', {});
+    securityTokenMock = ImportMock.mockClass(contractWrappersModule, 'SecurityToken_3_0_0');
+    tokenFactoryMockStub = tokenFactoryMock.mock(
```

Comment:
This stub doesn't seem to be used anywhere, so I don't see the need to assign it to a variable